### DISCUSSION TITLE
Lunch Options

### DISCUSSION THREAD

**Discussion by sqlworldwide** [dataplat/DataSaturdays#160] [Q&A]
- Bring your own lunch (yes/no)
- If No list different options to choose from

**Comment by lowlydba** [comment]
It would be neat to be able to say 'No' but 'I'll be going to restaurant xx and am open for others to join' with a counter for people to see how many have opted in to the ad-hoc offsite option (in NYC SQL Sat I almost always went out for food to get air + take advantage of the numerous non-sandwich options)

**Comment by sqlworldwide** [comment]
There are many events that happen that no good options available around the venue. Plus we hold sponsor sessions. That way attendees can grab lunch and listen to them. We charge a premium $$ for those sessions.

**Comment by lowlydba** [comment]
Yeah, I had a feeling NYC might be too much of an edge case 😢

**Comment by andrekamman** [comment]
One of the bugs that annoyed me the most on the SQLSat platform was that you couldn't switch off the lunch option. 
Many event locations have their own caterer, you can't bring your own lunch

[a comment by an automated account is not shown]
